graph/patterns/enums.md

```diff
@@ -1,25 +1,33 @@
-### Enums
+# Enums
 
-In OData, enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
+Microsoft Graph API Design Pattern
 
-```xml
-<EnumType Name="color">
-    <Member Name="Red" Value="0" />
-    <Member Name="Green" Value="1" />
-    <Member Name="Blue" Value="2" />
-</EnumType>
-```
+Enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
+
+## Problem
+
+The API Design requires that a specific property of a data structure has a finite and limited set of values, and such limitation needs to be clearly communicated to the user and give the developer the same structure
+
+## Solution
+
+In OData, Enums represent a subset of the nominal type they rely on, and are especially useful in cases where certain properties have predefined, limited options.
 
-#### Pros
+## When to use this pattern
+
+Everytime a specific property of a data structure has a finite and limited set of values, and such limitation needs to be clearly communicated to the user and give the developer the same structure.
+
+## Issues and considerations
+
+### Pros
 
 - Our SDK generators will translate the enum to the best representation of the target programming language, resulting in a better developer experience and free client side validation
 
-#### Cons
+### Cons
 
 - Adding a new value requires to go through a (generally fast) API Review

```

Comment:
Our internal process shouldn't impact API design considerations especially as they published for external users.

Reply:
Please remove this. We might offer some process guidance in the future, but I don't think it's our area of expertise now or something we are working on.